src/main/java/cloud/appuio/keycloak/extensions/mappers/GroupNameFormatter.java

```diff
@@ -0,0 +1,67 @@
+package cloud.appuio.keycloak.extensions.mappers;
+
+import org.keycloak.provider.ProviderConfigProperty;
+
+public class GroupNameFormatter {
+
+    private boolean trimWhitespace;
+    private boolean toLowerCase;
+    private String trimPrefix = "";
+
+    GroupNameFormatter() {
+    }
+
+    String format(String input) {
+        if (input == null) return "";
+        var s = input;
+        if (!trimPrefix.equals("")) {
+            s = s.replaceFirst(trimPrefix, "");
+        }
```

Comment:
Strictly speaking the condition is probably not required, as `replaceFirst("", "")` should be a no-op anyway.

```suggestion
        s = s.replaceFirst(trimPrefix, "");
```